### DISCUSSION TITLE
New camera zoomed in in bonsai, not other software.

### DISCUSSION THREAD

**Discussion by YUNoSol** [bonsai-rx/bonsai#1980] [Q&A]
Hi all

I have been using bonsai for a while now, so first of all, thank you for an amazing software!

But when I wanted to introduce a new camera(DMK 37AUX273) to the workflow, I encountered a curious problem, and I do not know how to solve it. When acquiring with the new camera in bonsai, it seems like the video is zoomed in (see screenshot 1). When I acquire with IC capture immediately after (same distance, same lens, without touching the focus), it looks normal and not zoomed in (see screenshot 2)

When I use the old cameras (DMK 37 AUX287) with the same lens it looks like screenshot 2. 
Do any of you have any ideas of what could have happened here? It seems to be something in bonsai and not the camera, since it works fine with IC capture.

The screenshots are from videos acquired with the simplest workflow of just CameraCapture -> VideoWriter.

Screenshot 1: 
![image](https://github.com/user-attachments/assets/04e17d39-e68d-45bc-9eca-97bc2116c415)
Screenshot2:
![image](https://github.com/user-attachments/assets/181a384f-a4b1-4819-845f-ac929a519b3c)
 
Thank you in advance!

Best regards, 
Sol

**Comment by bruno-f-cruz** [comment]
When using bonsai you are interfacing with the camera via the OpenCV backend (kudos to the manufacturer to actually enable it :P). As a result you will not be able to access all the settings the computer vision camera has to offer. I suspect that "zoom" is likely one of them. You can try using VideoCaptureDevice and check if it exposes additional settings by clicking the small top right icon:

![image](https://github.com/user-attachments/assets/d1496f3a-c7f7-4c55-b3bc-c774e5adbcc6)

Or playing with the `Format` property. Otherwise, it seems two of their SDKs are targeting .NET so it should be possible to integrate them in Bonsai too.

https://www.theimagingsource.com/en-us/support/download/icimagingcontrol4win-1.1.0.2833/
https://www.theimagingsource.com/en-us/support/download/icimagingcontrolcsharp-3.5.7/

**Reply by YUNoSol** [reply]
Hi Bruno

Thank you for the help once again.

Using VideoCaptureDevice worked like a charm! I just had to change the format property to the correct resolution, and then it looked like what I expected to see.

Have a great day.

Best regards, 
Sol
